src/test/java/com/axlabs/neo/grantshares/GrantSharesTreasuryTest.java

```diff
@@ -100,4 +98,57 @@ public void getFunders() throws IOException {
                 denise.getECKeyPair().getPublicKey().getEncoded(true)));
     }
 
-}
+    @Test
+    public void removedFunders() throws Throwable {
+
+        ContractParameter intents = array(
+                array(treasury.getScriptHash(), REMOVE_FUNDER,
+                        array(publicKey(ext.getAccount(BOB).getECKeyPair().getPublicKey().getEncoded(true)))));
+        String desc = "execute_proposal_with_remove_founder";
+
+        // 1. Create and endorse proposal
+        int id = createAndEndorseProposal(gov, neow3j, bob, alice, intents, desc);
+
+        // 2. Skip to voting phase and vote
+        ext.fastForward(PHASE_LENGTH);
+        voteForProposal(gov, neow3j, id, alice);
+
+        // 3. Skip till after vote and queued phase, then execute.
+        ext.fastForward(PHASE_LENGTH + PHASE_LENGTH);
+        Hash256 tx = gov.invokeFunction(EXECUTE, integer(id))
+                .signers(AccountSigner.calledByEntry(alice)
+                        .setAllowedContracts(treasury.getScriptHash()))
+                .sign().send().getSendRawTransaction().getHash();
+        Await.waitUntilTransactionIsExecuted(tx, neow3j);
+
+        NeoApplicationLog.Execution execution = neow3j.getApplicationLog(tx).send()
+                .getApplicationLog().getExecutions().get(0);
+        NeoApplicationLog.Execution.Notification n = execution.getNotifications().get(0);
+        assertThat(n.getEventName(), is("FunderRemoved"));
```

Comment:
Please also add a check here that retrieves the list of funders and makes sure that Bob is not present anymore.
```java
treasury.callInvokeFunction(GET_FUNDERS).getInvocationResult()
                .getStack().get(0).getList();
```